modules/AOS_PDF_Templates/vardefs.php

```diff
@@ -43,44 +43,54 @@
 }
 
 $dictionary['AOS_PDF_Templates'] = array(
-	'table'=>'aos_pdf_templates',
-	'audited'=>true,
-	'fields'=>array (
-  'active' =>
-  array (
-    'name' => 'active',
-    'vname' => 'LBL_ACTIVE',
-    'type' => 'bool',
-    'massupdate' => 0,
-    'default' => true,
-    'comments' => '',
-    'help' => '',
-    'importable' => 'true',
-    'duplicate_merge' => 'disabled',
-    'duplicate_merge_dom_value' => '0',
-    'audited' => 1,
-    'reportable' => 0,
-    'studio' => 'visible',
-  ),
-  'type' =>
-  array (
-    'required' => '1',
-    'name' => 'type',
-    'vname' => 'LBL_TYPE',
-    'type' => 'enum',
-    'massupdate' => 0,
-    'default' => '',
-    'comments' => '',
-    'help' => '',
-    'importable' => 'true',
-    'duplicate_merge' => 'disabled',
-    'duplicate_merge_dom_value' => '0',
-    'audited' => 1,
-    'reportable' => 0,
-    'len' => 100,
-    'options' => 'pdf_template_type_dom',
-    'studio' => 'visible',
-  ),
+    'table' => 'aos_pdf_templates',
+    'audited' => true,
+    'fields' => array(
+        'description' =>
+            array(
+                'name' => 'description',
+                'vname' => 'LBL_DESCRIPTION',
+                'type' => 'text',
+                'comment' => 'Full PDF text',
```

Comment:
previous #1733 has the comment field translated, this has not
`                'comment' => 'LBL_PDF_COMMENT',`
See
https://github.com/salesagility/SuiteCRM/pull/3532/files#diff-58f1445ded3b32a935750f69b3c61e0aR54